src/shared/defineFeatured.ts

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable max-len */
+import type {SIM} from './browserCaching'
+
+// Use the array below to define the featured specimens.
+// Note that `en64` is the base64 encoding of the desired specimens, which
+// can be found as the value of the `specimen=` query string in the URL.
+
+export function getFeatured(): SIM[] {
+    const featuredSIMs = [
+        {
+            en64: 'eyJuYW1lIjoiQmFzaWMgTW9kZmlsbCIsInNlcXVlbmNlIjoiUmFuZG9tIiwic2VxdWVuY2VQYXJhbXMiOiJleUp0YVc0aU9pSXlJaXdpYldGNElqb2lNVFFpZlE9PSIsInZpc3VhbGl6ZXIiOiJNb2RGaWxsIiwidmlzdWFsaXplclBhcmFtcyI6ImV5SnRiMlJFYVcxbGJuTnBiMjRpT2lJeE1pSjkifQ==',
+            name: 'Basic Modfill',
+            date: '',
+        },
+        {
+            en64: 'eyJuYW1lIjoiVHdlbHZlIiwic2VxdWVuY2UiOiJSYW5kb20iLCJzZXF1ZW5jZVBhcmFtcyI6ImV5SnRhVzRpT2lJeE1pSXNJbTFoZUNJNklqRXlJbjA9IiwidmlzdWFsaXplciI6Ik1vZEZpbGwiLCJ2aXN1YWxpemVyUGFyYW1zIjoiZXlKdGIyUkVhVzFsYm5OcGIyNGlPaUl4TWlKOSJ9',
+            name: 'Twelve',
+            date: '',
+        },
+    ]
```

Comment:
Here's another nice one:
```
        {
            en64: 'eyJuYW1lIjoiV2FpdCBGb3IgSXQiLCJzZXF1ZW5jZSI6IkZvcm11bGEiLCJzZXF1ZW5jZVBhcmFtcyI6ImV5Sm1iM0p0ZFd4aElqb2liV2x1S0dac2IyOXlLSE5wYmlodUtTa3BKVFVpZlE9PSIsInZpc3VhbGl6ZXIiOiJUdXJ0bGUiLCJ2aXN1YWxpemVyUGFyYW1zIjoiZXlKa2IyMWhhVzRpT2lJd0xDQXhMQ0F5TENBekxDQTBJaXdpY21GdVoyVWlPaUl6TUN3Z05EVXNJRFl3TENBNU1Dd2dNVEl3SWl3aWMzUmxjRk5wZW1VaU9pSXpNQ0lzSW5OMFlYSjBJam9pTUN3Z01DSXNJbk4wY205clpWZGxhV2RvZENJNklqSWlMQ0ppWjBOdmJHOXlJam9pSXpWa05UQTVaaUlzSW5OMGNtOXJaVU52Ykc5eUlqb2lJemRoT1dZMlppSjkifQ==',
            name: 'Wait For It',
            date: '',
        }, 
```

Reply:
OK, added that one, too, in the latest commit. Nice. This is something about τ and 1 being incommensurable? What's the `min` in there do?

Reply:
Oh, oops, the min is left over from some experiment.

Reply:
OK, I will take the `min` out, regrab the base64,and put the simpler one in the defineFeatured.

Reply:
Here's [the version without min](eyJuYW1lIjoiV2FpdCBGb3IgSXQiLCJzZXF1ZW5jZSI6IkZvcm11bGEiLCJzZXF1ZW5jZVBhcmFtcyI6ImV5Sm1iM0p0ZFd4aElqb2labXh2YjNJb2MybHVLRzRwS1NVMUluMD0iLCJ2aXN1YWxpemVyIjoiVHVydGxlIiwidmlzdWFsaXplclBhcmFtcyI6ImV5SmtiMjFoYVc0aU9pSXdMQ0F4TENBeUxDQXpMQ0EwSWl3aWNtRnVaMlVpT2lJek1Dd2dORFVzSURZd0xDQTVNQ3dnTVRJd0lpd2ljM1JsY0ZOcGVtVWlPaUl6TUNJc0luTjBZWEowSWpvaU1Dd2dNQ0lzSW5OMGNtOXJaVmRsYVdkb2RDSTZJaklpTENKaVowTnZiRzl5SWpvaUl6VmtOVEE1WmlJc0luTjBjbTlyWlVOdmJHOXlJam9pSXpkaE9XWTJaaUo5In0) (same behaviour).
```
eyJuYW1lIjoiV2FpdCBGb3IgSXQiLCJzZXF1ZW5jZSI6IkZvcm11bGEiLCJzZXF1ZW5jZVBhcmFtcyI6ImV5Sm1iM0p0ZFd4aElqb2labXh2YjNJb2MybHVLRzRwS1NVMUluMD0iLCJ2aXN1YWxpemVyIjoiVHVydGxlIiwidmlzdWFsaXplclBhcmFtcyI6ImV5SmtiMjFoYVc0aU9pSXdMQ0F4TENBeUxDQXpMQ0EwSWl3aWNtRnVaMlVpT2lJek1Dd2dORFVzSURZd0xDQTVNQ3dnTVRJd0lpd2ljM1JsY0ZOcGVtVWlPaUl6TUNJc0luTjBZWEowSWpvaU1Dd2dNQ0lzSW5OMGNtOXJaVmRsYVdkb2RDSTZJaklpTENKaVowTnZiRzl5SWpvaUl6VmtOVEE1WmlJc0luTjBjbTlyWlVOdmJHOXlJam9pSXpkaE9XWTJaaUo5In0
```

Reply:
Actually I changed the formula to `sign(sin(n+1))` and put only -1 and 1 in the domain; I think that makes what is going on clearer. (Sadly the n+1 is necessary because Turtle always starts at term 0 and sign(sin(0)) is 0, the only integer for which that is true.

Reply:
looks good

Reply:
looks good